### PR TITLE
Fixup some nits from #44238

### PR DESCRIPTION
**Pull request by nrc**
r? @ollie27

### CODE REVIEW

**Comment by nrc** [comment]
cc https://github.com/rust-lang/rust/issues/44229

File: src/librustdoc/html/render.rs [hunk 1]
```diff
@@ -606,12 +606,20 @@ pub fn run(mut krate: clean::Crate,
...
 // A short, single-line view of `s`.
-fn concise_str(s: &str) -> String {
+fn concise_str(mut s: &str) -> String {
     if s.contains('\n') {
-        return format!("{}...", s.lines().next().expect("Impossible! We just found a newline"));
+        s = s.lines().next().expect("Impossible! We just found a newline");
     }
     if s.len() > 70 {
-        return format!("{} ... {}", &s[..50], &s[s.len()-20..]);
+        let mut lo = 70;
```
Comment (ollie27) [inline src/librustdoc/html/render.rs]: This should be 45 to make sure the resulting string is at most 70 characters.

**Review by ollie27 — COMMENTED** [review]
Apart from that one error this looks good to me.

File: src/librustdoc/html/render.rs (hunk 1 above)
Comment (nrc) [reply src/librustdoc/html/render.rs]: Addressed this, thanks! I set to 50 (as in the old code), there is no strict need to be under 70 chars, it is just roughly a number that should fit on one line (comfortably under 80 chars).

Comment (ollie27) [reply src/librustdoc/html/render.rs]: Well that's okay but if the line is exactly 71 characters for example then using 50 will skip one character and replace it with `" ... "` making the string display actually longer. It's only a minor thing though.

**Comment by nrc** [comment]
ping @ollie27 for re-review

**Pull request merged**